python/py-pythran/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-pythran
-version             0.14.0
+version             0.15.0
 revision            1
```

Comment:
the "revision" should be reset to "0" when updating the version

Reply:
Thanks, I forgot that. I pushed a new commit with both suggestions.